regolith/tools.py

```diff
@@ -406,24 +406,17 @@ def fuzzy_retrieval(documents, sources, value, case_sensitive=True):
 
     """
     for doc in documents:
-        fs = []
-        for source in sources:
-            # TODO: This needs to be nested get
-            # (so source can be a tuple of keys)
-            v = doc.get(source, [])
-            if isinstance(v, list):
-                fs.extend(v)
-            else:
-                fs.append(v)
         returns = []
         for k in sources:
             ret = doc.get(k, [])
             if isinstance(ret, str):
                 returns.append(ret)
+            elif isinstance(ret, int):
```

Comment:
this most likely should be combined with the above so if it is not a list then it makes it into a list and extends returns by it.

Reply:
how about:
```
if not isinstance(ret, list):
    ret = [ret]
elif ret:
    returns.extend(ret)
```

Reply:
I think we can remove the `elif ret:` and de-indent the `returns.extend(ret)` since we want to extend the returns with ret.

Reply:
yes, it is not needed because we set up get to return [] if it can't find k.

OK, look for the next push.